src/server/list_family_test.cc

```diff
@@ -793,13 +793,20 @@ TEST_F(ListFamilyTest, BLMoveSimultaneously) {
 
 TEST_F(ListFamilyTest, BLMoveRings) {
   vector<fb2::Fiber> fibers;
+  fibers.reserve(10);
+#pragma GCC diagnostic push
+// We compile this code both with C++17 and C++20 and if you capture
+// by [=, this] it becomes an error on C++17 and if you capture
+// by [=] it becomes and error in C++20
+#pragma GCC diagnostic ignored "-Wdeprecated"
```

Comment:
we should post this comment to some r/cpprant or rust subreddit 😆

Reply:
Dude you have no idea how annoyed I was